### PR TITLE
Update perms  for mr.shifty

### PR DESCRIPTION
**Pull request by ThisChicken**
I fixed him in the second update, please fix

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -1102,7 +1102,7 @@ groups:
         - ch.alias.haste0
         - ch.alias.haste1
         - ch.alias.haste2
-  mrshifty:
+  mr.shifty:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: remove the .

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -1113,6 +1113,8 @@ groups:
         permissions:
         - ch.alias.mr.shifty
         - -essentials.protect.damage.fall
+        - worldedit.navigation.thru.command
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: please show evidence of intangibility

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -1113,6 +1113,8 @@ groups:
...
+        - worldedit.navigation.jumpto.command
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: please show evidence of TP to anywhere in sight

Comment (ThisChicken) [inline PermissionsEx/permissions.yml]: in this gif, he is seen teleporting where ever he sees : https://www.gamingonlinux.com/uploads/articles/article_images/1472649950Destruction.gif

File: PermissionsEx/permissions.yml (hunk 2 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: approved

File: PermissionsEx/permissions.yml (hunk 3 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: that is extremely limited..can you show him TPing farther than that?

Comment (ThisChicken) [inline PermissionsEx/permissions.yml]: he can only go as far as he can see, or for the game's sake, as far as the little dot is so he cant skip entire rooms.

Comment (malonnnn) [inline PermissionsEx/permissions.yml]: giving him ender pearls would be fine, but even ender pearls go farther than 2 feet

**Pull request merged**